### DISCUSSION TITLE
16bit colour table shows some strange values

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#936] [Feature Requests] [answered]
Bug:

I have a red gradient going from 0-1 which you can see in the 32 bit colour texture table. The 16bit tables shows some strange values... Should be 0-1? 

https://cables.gl/edit/Xo9Uqv 

<img width="986" alt="Screenshot 2024-11-28 at 15 57 17" src="https://github.com/user-attachments/assets/0e9b1744-329d-474b-9c70-c8b65ed9f7f9">

likely why I'm having trouble picking on mobile.
https://cables.gl/edit/mXlSev

**Comment by pandrr** [comment, marked as answer]
moved to issue https://github.com/cables-gl/cables_docs/issues/937

**Reply by pandrr** [reply]
thannks for reporting
fixed on dev.cables.gl

if clearly a bug, please create an issue, not discussion.
